frontend/packages/dev-console/src/utils/shared-submit-utils.ts

```diff
@@ -78,6 +78,13 @@ export const createService = (
     const port = { containerPort: _.toInteger(unknownTargetPort), protocol: 'TCP' };
     ports = [...ports, port];
   }
+  if (
+    unknownTargetPort &&
+    unknownTargetPort !== '8080' &&
+    ports.some((port) => port.containerPort.toString() === '8080')
+  ) {
+    ports = ports.filter((port) => port.containerPort.toString() !== '8080');
```

Comment:
Would this remove ports from existing services? Not sure if we should be removing ports here, especially in edit flow.